setup.py

```diff
@@ -93,15 +93,15 @@ def get_transforms_data():
         'astropy>=4.0',
         'crds>=7.2.7',
         'drizzle>=1.13',
-        'gwcs>=0.12',
+        'gwcs @ git+https://github.com/spacetelescope/gwcs@1c1cb3bb35caddef80fb760ea68bc71e189d32de',
         'jsonschema>=2.3,<4',
         'numpy>=1.16',
         'photutils>=0.7',
         'scipy>=1.0',
         'spherical-geometry>=1.2',
         'stsci.image>=2.3.3',
         'stsci.imagestats>=1.4',
-        'tweakwcs>=0.5.3',
+        'tweakwcs @ git+https://github.com/spacetelescope/tweakwcs@553ebd51262ca980c58aaf0a980c9cf746c08bb4',
```

Comment:
This should be changed to `tweakwcs>=0.6`?  Or whichever minimum version is appropriate.